__tests__/unit/utils.test.js

```diff
@@ -1,23 +1,124 @@
 import search from '../../src/utils/search'
 
+const matrix = [
```

Comment:
to make sure we have consistent indentation across the project, we can add the test directory into our linting directories by doing:
add ` __tests__` at the end of line 12 & 13 of package.json
```
    "lint": "eslint src stories __tests__",
    "lint:watch:fix": "esw --fix --watch src stories __tests__",
```
And
add `    'jest': true,` to line 6 of `.eslintrc`
at the end, run `yarn lint:watch:fix` to auto fix all lint errors.

Reply:
additional comment on `yarn`. parameter pass-through is built-in (unlike `npm`), you can do `yarn lint --fix` directly (versus `npm run lint -- --fix`, where an explicit `--` in the middle is needed for parameter pass-through)